src/software/ai/hl/stp/play_info.h

```diff
@@ -3,22 +3,57 @@
 #include <string>
 #include <vector>
 
-struct PlayInfo
+class PlayInfo
 {
+   public:
+    /**
+     * Constructs a PlayInfo object.
+     */
+    explicit PlayInfo();
+
+    /**
+     * @return play_type field
+     */
```

Comment:
Generally I'd prefer javadocs look like so: 
```
/**
 * Gets play type (high level comment on what the function does)
 *
 * @return string representing the play type (return type and what it represents)
 */
```
But I understand that for getters, it's self-evident